draft-ietf-core-senml.md

```diff
@@ -718,63 +726,77 @@ number of this specification.
+| lx       | lux                                               | float | RFC-AAAA  |
+| Bq       | becquerel                                         | float | RFC-AAAA  |
+| Gy       | gray                                              | float | RFC-AAAA  |
+| Sv       | sievert                                           | float | RFC-AAAA  |
+| kat      | katal                                             | float | RFC-AAAA  |
+| m2       | square meter (area)                               | float | RFC-AAAA  |
+| m3       | cubic meter (volume)                              | float | RFC-AAAA  |
+| l        | liter (volume)*                                   | float | RFC-AAAA  |
+| m/s      | meter per second (velocity)                       | float | RFC-AAAA  |
+| m/s2     | meter per square second (acceleration)            | float | RFC-AAAA  |
+| m3/s     | cubic meter per second (flow rate)                | float | RFC-AAAA  |
+| l/s      | liter per second (flow rate)*                     | float | RFC-AAAA  |
+| W/m2     | watt per square meter (irradiance)                | float | RFC-AAAA  |
+| cd/m2    | candela per square meter (luminance)              | float | RFC-AAAA  |
+| bit      | bit (information content)                         | float | RFC-AAAA  |
+| B        | Byte (information content, note 3)                | float | RFC-AAAA  |
+| bit/s    | bit per second (data rate)                        | float | RFC-AAAA  |
+| lat      | degrees latitude (note 2)                         | float | RFC-AAAA  |
+| lon      | degrees longitude (note 2)                        | float | RFC-AAAA  |
+| pH       | pH value (acidity; logarithmic quantity)          | float | RFC-AAAA  |
+| dB       | decibel (logarithmic quantity)                    | float | RFC-AAAA  |
+| Bspl     | bel (sound pressure level; logarithmic quantity)* | float | RFC-AAAA  |
+| count    | 1 (counter value)*                                | float | RFC-AAAA  |
+| %        | Percentage (Value of a switch, note 1)*           | float | RFC-AAAA  |
+| %RH      | Percentage (Relative Humidity)*                   | float | RFC-AAAA  |
+| %EL      | Percentage (remaining battery energy level)*      | float | RFC-AAAA  |
+| EL       | seconds (remaining battery energy level)*         | float | RFC-AAAA  |
+| 1/s      | 1 per second (event rate)                         | float | RFC-AAAA  |
+| 1/min    | 1 per minute (event rate)*                        | float | RFC-AAAA  |
+| beat/min | 1 per minute (Heart rate in beats per minute)*    | float | RFC-AAAA  |
+| beats    | 1 (Cumulative number of heart beats)*             | float | RFC-AAAA  |
 {: #tbl-iana-symbols cols='r l l'}
 
-* Note 1: A value of 0.0 indicates the switch is off while 1.0
-  indicates on and 0.5 would be half on. 
+* Note 1: A value of 0.0 indicates the switch is off while 100.0
+  indicates on and 50.0 would be half on.
```

Comment:
> On Jul 7, 2016, at 3:27 PM, cabo <notifications@github.com> wrote:
> 
> In draft-ietf-core-senml.md <https://github.com/core-wg/senml-spec/pull/6#discussion_r69999244>:
> 
> >  {: #tbl-iana-symbols cols='r l l'}
> >  
> > -* Note 1: A value of 0.0 indicates the switch is off while 1.0
> > -  indicates on and 0.5 would be half on. 
> > +* Note 1: A value of 0.0 indicates the switch is off while 100.0
> > +  indicates on and 50.0 would be half on.
> Oh. Keeping % to mean something else but a percentage creates exactly the interoperability problem I was afraid of. This requires a big warning. How can one possibly use % for 0..1...
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/core-wg/senml-spec/pull/6/files/5d8e3730c6b18fadd1c0263488a0075707c76a60..11ef3ac9a8214dcb49a133bea97f5445a6d2c849#r69999244>, or mute the thread <https://github.com/notifications/unsubscribe/AANQcFMLYEBrm2yfTdRvhrR-4PCAX3KHks5qTX1vgaJpZM4JHTeM>.
> 

yah - i agree this is a huge mess. I wonder if the best path would be defining a new “/“ or “ratio” that ranges from 0 to 1 and replaces “%” and we mark “%” with the don’t use *. Even if we stay with % and 0 to 1, or change to 0 to 100, we need a huge warning label.